xbmc/filesystem/VideoDatabaseDirectory/DirectoryNodeTitleMovies.cpp

```diff
@@ -28,7 +28,14 @@ bool CDirectoryNodeTitleMovies::GetContent(CFileItemList& items) const
   CQueryParams params;
   CollectQueryParams(params);
 
-  bool bSuccess=videodatabase.GetMoviesNav(BuildPath(), items, params.GetGenreId(), params.GetYear(), params.GetActorId(), params.GetDirectorId(), params.GetStudioId(), params.GetCountryId(), params.GetSetId(), params.GetTagId());
+  int details = items.HasProperty("set_videodb_details")
```

Comment:
guess it is fine to keep the return type `int` while `CVariant::asInteger32` returns `int32_t`

Reply:
and we've seen that `asInteger()` returns `int64_t`...and `int32_t` is cast from `int64_t` too
The beauty of Kodi :)